PowerEditor/src/Parameters.h

```diff
@@ -279,7 +281,9 @@ struct CmdLineParamsDTO
 	intptr_t _pos2go = 0;
 
 	LangType _langType = L_EXTERNAL;
-	generic_string _udlName;
+
+	wchar_t _udlName[MAX_PATH];
+	wchar_t _pluginMessage[MAX_PATH];

```

Comment:
The previous version of `generic_string _udlName;` has to be converted into `wchar_t _udlName[MAX_PATH]` to prevent the crash reported in issue #11633.